nomenclature/region_mapping_models.py

```diff
@@ -25,28 +26,29 @@ class CommonRegion(BaseModel):
 
 class RegionAggregationMapping(BaseModel):
     model: str
+    mapping_file: FilePath
```

Comment:
Suggest to name that attribute just `file` in line with the codelist structure, see https://github.com/IAMconsortium/nomenclature/blob/72a142236050f6bda0ee4f80b96f77e8b17ea52b/nomenclature/codes.py#L122

Reply:
Done